### PR TITLE
* modified:   pages/dmp_supporting_info/support_00_planning_considera…

### PR DESCRIPTION
**Pull request by jennyostrop**
…tions.md:	changed section order and improvements after tester feedback, updated KM reference

### CODE REVIEW

**Comment by jennyostrop** [comment]
Please see over and approve or suggest changes, that I subsequently can commit. Merging requires an approved review.

File: pages/dmp_supporting_info/support_00_planning_considerations.md [hunk 1]
```diff
@@ -137,4 +122,32 @@ If the project plans to recruit or train dedicated staff, related costs should b
 #### Will there be costs related to research data management?
 Will research data management (RDM) in the project require additional resources? Consider costs for data storage, data processing, data archiving, expert support.
 
-Many research funders consider list research data management as eligible costs in applications. Have costs for RDM been included in the project budget?
\ No newline at end of file
+Many research funders consider list research data management as eligible costs in applications. Have costs for RDM been included in the project budget?
```
Comment (sveinho) [inline pages/dmp_supporting_info/support_00_planning_considerations.md:125]: choose to use consider or list in stead of both here?

File: pages/dmp_supporting_info/support_00_planning_considerations.md [hunk 2]
```diff
@@ -137,4 +122,32 @@ If the project plans to recruit or train dedicated staff, related costs should b
...
+
+Please consult [RDMkit and OpenAIRE resources](support_00_planning_considerations#further-resources)for further information on RDM cost calculations.
+
+
+
+## Addressing research data management in grant proposals
+Some research funders ask about Open Science and/ or data management planning already in grant applications. In this section you should demonstrate that you have reflected on how your research results can be made available including some practical aspects and how this will strengthen the impact of your project. You are not required to have a complete DMP at the proposal stage.
```
Comment (sveinho) [inline pages/dmp_supporting_info/support_00_planning_considerations.md:132]: Suggestion: You are, however, not required to have...

**Review by sveinho — APPROVED** [review]
Fungerer kjempefint, dette. To små språklige kommentarer, that is it. Trykker approve.

**Pull request merged**